.goreleaser.draft.yml

```
@@ -39,7 +39,8 @@ archives:
 sboms:
   - artifacts: archive
     documents:
-      - "{{ .ProjectName }}_{{ .Version }}_sbom.spdx.json"
+      - "binary":   ["{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}.sbom"]
```

```suggestion
      - binary:   ["{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}.sbom"]
```

Also ensure to apply same on the `.goreleaser.yml`. Preferably keep them in sync in the same commit.

---

I think you reviewed, just when I pushed a new commit fixing this. Can you re-review?